Sources/StreamChatUI/MessageActionsPopup/ChatMessageActionsVC.swift

```diff
@@ -88,6 +88,9 @@ open class _ChatMessageActionsVC<ExtraData: ExtraDataTypes>: _ViewController, UI
                 threadReplyActionItem(),
                 copyActionItem()
             ]
+            
+            let threadReply = threadReplyActionItem()
+            threadReply.action(threadReply)
```

Comment:
what's happening here? 🤯

Reply:
It was a temp workaround for testing thread replies since message actions are broken